data/human/jobs.txt

```diff
@@ -3899,3 +3899,985 @@ mission "Republic Navy Advisory System [wildfire firefighters]"
 	on complete
 		payment 9000 85
 		dialog "You land at a makeshift landing site a few minutes flight from the blaze. The smoky haze that drifts into your ship when the firefighters open the hatch to depart makes your eyes and nose itch. As you watch your passengers heading toward a staging area nearby, you think about the inferno they will soon be voluntarily entering. A chirp from your communications system indicates your stipend of <payment> has been deposited, and you begin your atmospheric flight back to <planet>'s main spaceport."
+
+mission "Syndicate Defectors [0]"
+	name "Syndicate defectors to <planet>"
+	job
+	repeat
+	description "Go to <stopovers> and pick up <bunks> Syndicate defectors. Fly them to <destination> for <payment>. Expect bounty hunters."
+	passengers 2 8 .8
+	clearance "You provide a passcode given to you on <origin> and the authorities give you landing coordinates."
+	to offer
+		random < 60
+	source
+		government "Independent (Korath-Allied) Port"
+	destination
+		government "Independent (Korath-Allied) Port"
+	stopover
+		government "Pirate"
```

Comment:
What are syndicate defectors doing on pirate planets?

Reply:
Lawless places like pirate ports are great places for people who don't particularly want to be found by people in authority.

Reply:
In that case, the job should be `illegal.`

Reply:
All "Syndicate Defector" missions are now marked illegal. The first one has a comment that the illegality awaits a bug fix.